examples/ember-cli/app/components/welcome-banner.js

```diff
@@ -1,50 +1,50 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 
 /**
  *
  * `WelcomeBanner` renders a friendly message and is used to welcome Ember.js users when they first generate an application.
  *
  *
  * ```js
- * {{welcome-banner
- *   backgroundColor=backgroundColor
- *   titleColor=titleColor
- *   subTitleColor=subTitleColor
- *   title=title
- *   subtitle=subtitle
- *   click=(action onClick)
- * }}
+ * <WelcomeBanner
+ *   @backgroundColor={{backgroundColor}}
+ *   @titleColor={{titleColor}}
+ *   @subTitleColor={{subTitleColor}}
+ *   @title={{title}}
+ *   @subtitle={{subtitle}}
+ *   @click={{(action onClick)}}
+ * />
  * ```
  *
  * @class WelcomeBanner
  */
-export default Component.extend({
+export default class WelcomeBanner extends Component {
   /**
    * The hex-formatted color code for the background.
    * @argument backgroundColor
    * @type {string}
    * @default null
    */
-  backgroundColor: null,
+  backgroundColor = null;
 
   /**
    * The hex-formatted color code for the subtitle.
    * @argument subTitleColor
    * @type {string}
    */
-  subTitleColor: null,
+  subTitleColor = null;
 
   /**
    * The title of the banner.
    * @argument title
    * @type {string}
    */
-  title: null,
+  title = null;
 
   /**
    * The subtitle of the banner.
    * @argument subtitle
    * @type {string}
    */
-  subtitle: null,
-});
+  subtitle = null;
```

Comment:
this is no longer valid code for glimmer components. In typescript you would put this into an `WelcomeBannerArgs` interface and then do `export default class WelcomeBanner extends Component<WelcomeBannerArgs>`.